enterprise_access/apps/api_client/tests/test_license_manager_client.py

```diff
@@ -5,13 +5,20 @@
 from unittest import mock
 
 from django.conf import settings
-from django.test import TestCase
+from django.test import RequestFactory, TestCase
+from faker import Factory as FakerFactory
 from requests import Response
 
-from enterprise_access.apps.api_client.license_manager_client import LicenseManagerApiClient
+from enterprise_access.apps.api_client.constants import ACTIVATED_LICENSE
+from enterprise_access.apps.api_client.license_manager_client import (
+    LicenseManagerApiClient,
+    LicenseManagerUserApiClient
+)
+from enterprise_access.apps.api_client.tests.test_constaints import DATE_FORMAT_ISO_8601, DATE_FORMAT_ISO_8601_MS
+from enterprise_access.apps.content_assignments.tests.test_utils import _days_from_now
```

Comment:
we should refactor `_days_from_now()` into a public function that's available in a common module - the `enterprise_access/utils.py` file would be a good place.